### PR TITLE
README: add gif to show installation

### PR DESCRIPTION
**Pull request by vrothberg**
To help users install the extension, add a gif that shows the workflow. Also add a small description of the extension to the top.

I want to add more gifs in the future but at the moment there are many things in motion, so I prefer to wait a couple of days.

### CODE REVIEW

**Review by benoitf — COMMENTED** [review]
FYI, for large files we put them into a separate repository https://github.com/containers/podman-desktop-media/ repository (for example there is a bootc branch https://github.com/containers/podman-desktop-media/tree/bootc-extension)

it avoids to have to clone large repository

File: README.md [hunk 1]
```diff
@@ -1,26 +1,16 @@
 # AI studio
 
-## Installing a released version
+The Podman Desktop AI Studio extension simplifies getting started and developing with AI in a local environment.  It provides key open-source technologies to start building on AI.  A curated catalog of so-called recipes helps navigate the jungle of AI use cases and AI models.  AI Studio further ships playgrounds: environments to test and refine AI models, for instance, a chat bot.
```
Comment (jeffmaury) [inline README.md:3]: Refining models is not part of the playground user experience, only playing and testing models

**Comment by vrothberg** [comment]
Thanks! Shall I move the gif to this repository?

**Comment by vrothberg** [comment]
It's 812k but I can move it if needed.

**Review by jeffmaury — COMMENTED** [review]
Seems the GIF does not show the AI Studio UI is that on purpose ?

**Comment by benoitf** [comment]
when adding images to README files, it needs to use absolute links and not relative links else yes it won't show up

@vrothberg I would say yes as probably more gifs will come and we'll replace the file over time

File: README.md (hunk 1 above)
Comment (vrothberg) [reply README.md:3]: I've seen mockups showing to parametrize the models in the playgrounds.  Are there still plans to do that?

**Comment by vrothberg** [comment]
@benoitf I only see mp4 files in https://github.com/containers/podman-desktop-media/tree/bootc-extension/videos.  The gifs are part of the bootc-extension (https://github.com/containers/podman-desktop-extension-bootc/tree/main/docs/img).

I think we only need gifs for the README but can follow the same pattern.  Do you know why there are mp4 and gif files?

**Comment by benoitf** [comment]
@vrothberg I would say that usually gifs are for small animation (few seconds) so if you miss something it's ok as it's repeating again and again

videos are useful to be able to pause (when it's taking like 1mn to display things) (because most of the people are not able to pause gifs))

**Comment by benoitf** [comment]
feel free to push files to https://github.com/containers/podman-desktop-media/tree/ai-lab

**Comment by vrothberg** [comment]
Resolved both comments ✅

**Comment by vrothberg** [comment]
> Seems the GIF does not show the AI Studio UI is that on purpose ?

That's because the dashboard is empty at the present.  I will add more gifs as we progress.

**Review by jeffmaury — APPROVED** [review]
LGTM

**Review by benoitf — APPROVED** [review]

**Pull request merged**